spec/requests/users_spec.rb

```diff
@@ -25,4 +25,21 @@
       end
     end
   end
+  path '/{locale}/check_username?name={username}' do
+    parameter name: 'locale', in: :path, type: :string, description: 'locale'
+    parameter name: 'username', in: :path, type: :string, description: 'username'
+
+    get('check username') do
+      tags 'Users'
+      produces 'application/json'
+      response(200, 'successful') do
+        schema type: :object,
+               properties: {
+                 username_exists: { type: :boolean }
```

Comment:
let's return to frontend side JSON with filds in camel case - they will use them and it will be more confortable at FE side:

'username_exists' -> 'usernameExists'

Are you agree with this convention?